tasks/push-to-cdn/push-to-cdn.yaml

```diff
@@ -0,0 +1,140 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: push-to-cdn
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to push content to Red Hat's CDN
+  params:
+    - name: dataPath
+      type: string
+      description: >
+        Path to the JSON string of the merged data to use in the data workspace.
```

Comment:
Nit: Other descriptions don't end with `.`, so this shouldn't either.

Reply:
update, ty!